app/controllers/users.rb

```diff
@@ -0,0 +1,55 @@
+class Chitter < Sinatra::Base
+  get '/users/new' do
+    @user = User.new
+    erb :'/users/new'
+  end
+
+  post '/users' do
+    @user = User.create(name: params[:name],
+                      username: params[:username],
+                      email: params[:email],
+                      password: params[:password],
+                      password_confirmation: params[:password_confirm])
+    if @user.id
+      session[:user_id]=@user.id
+      redirect '/'
+    else
+      flash.now[:error]=@user.errors.full_messages
+      erb :'/users/new'
+    end
+  end
+
+  get '/users/recover' do
+    erb :'/users/recover'
+  end
+
+  post '/users/recover' do
+    user = User.first(email: params[:email])
+    if user
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)